packages/bot-web-ui/src/app/__tests__/dbot-providers.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { mockStore, useStore } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render, screen } from '@testing-library/react';
+import { mock_ws } from '../../utils/mock';
+import DBotProviders from '../dbot-providers';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+}));
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
```

Comment:
const skeleton_scratch_path = '@deriv/bot-skeleton/src/scratch';
```suggestion
jest.mock(`${skeleton_scratch_path}/blockly`, () => jest.fn());
jest.mock(`${skeleton_scratch_path}/dbot`, () => ({
    saveRecentWorkspace: jest.fn(),
    unHighlightAllBlocks: jest.fn(),
}));
jest.mock(`${skeleton_scratch_path}/hooks/block_svg`, () => jest.fn());
```

Reply:
@maryia-matskevich-deriv I am unsure if we can add a variable as a path name using string concatenation. Please let me know if this works. I also tried now but it isn't working for me.

Reply:
<img width="1379" alt="image" src="https://github.com/binary-com/deriv-app/assets/129021108/9ce4a17e-8f7a-4f9f-b52c-f4e31420c8f4">

Reply:
yees, curious thing path with jest.mock(), even with var variable is undefined 😳 
I have checked a lot of cases on how to fix it, but no one works, It would be interesting to know how to work flexibly with paths in tests, it seems impossible at the moment.  If you find out how to let me know, I'm incredibly interested.

Reply:
Surely I will!!